helm-chart/renku-notebooks/values.yaml

```diff
@@ -101,6 +101,10 @@ serverOptions:
   defaultUrl:
     default: /lab
     options: [/lab]
+  lfs_skip_smudge:
```

Comment:
These changes won't be picked up by the UI. So far I assumed that the top-level properties are only `defaultUrl` and `resources`, and I display all options under `resources` automatically. If you really want this to be a configurable option through the UI, add the `enum` type to the `lsf_skip_smudge` object and move it into `resources`. In general I would prefer a flat structure for the `server_options` object as long as the total number of options is manageable.

Reply:
what about 

```
serverOptions:
  cpu_request:
    displayName: Number of CPUs
    type: integer
    default: 1
    range: [1, 8]
  mem_request:
    displayName: Memory
    type: enum
    default: 2G
    options: [2G, 4G, 8G]
  gpu:
    displayName: Number of GPUs
    type: integer
    default: 0
    range: [0, 4]
  defaultUrl:
    default: /lab
    options: [/lab]
  lfs_skip_smudge:
    default: 1
    options: [0,1]
    displayName: Do not automatically fetch data
```

Reply:
Yes, this would make the ui a bit simpler.

Reply:
I had `resources` in there initially to be a bit closer to the k8s pod spec, but I need to do the mapping by hand anyway so it's not very useful.